string name
float64 llc_row
float64 llc_col
float64 height
float64 width
